src/startHtml.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * start_html
+ *
+ * This plugin redacts all HTML tags and content in a start_html level string,
+ * such that the translator only sees the relevant text to translate.
+ *
+ * @example
+ * Source: <label
+ *          style="margin: 0px; padding: 2px;
+ *              line-height: 1; font-size: 40px;
+ *              overflow: hidden; word-wrap: break-word;
+ *              color: rgb(51, 51, 51);
+ *              max-width: 320px; width: 320px; height: 45px;
+ *              position: absolute; left: 0px; top: 15px;
+ *              text-align: center;"
+ *           id="label3">
+ *           Success!
+ *         </label>
+ * Redacted: [][0]Success![][1]
+ */
+
+let redact;
+const HTML_TAG_RE = /<\/{0,1}[a-zA-Z]+(>|.*?[^?]>)/;
+const START_HTML = 'startHtml';
+
+module.exports = function startHtml() {
+  if (this.Parser) {
+    const Parser = this.Parser;
+    redact = Parser.prototype.options.redact
+
+    Parser.prototype.inlineTokenizers[START_HTML] = tokenizeStartHtml;
+
+    // redact start_html before the parser tries to parse text as HTML
+    const methods = Parser.prototype.inlineMethods;
+    methods.splice(methods.indexOf('html'), 0, START_HTML);
+  }
+};
+
+tokenizeStartHtml.notInLink = true;
+tokenizeStartHtml.locator = locateStartHtml;
+
+function tokenizeStartHtml(eat, value, silent) {
+  const match = HTML_TAG_RE.exec(value);
+
+  if (!match || value.indexOf(match[0]) != 0) {
+    return;
+  }
+
+  if (silent) {
+    return true;
+  }
+
+  console.log("match[0]: " + match[0]);
+  console.log("value: " + value);
+  const add = eat(match[0]);
```

Comment:
Is `eat` basically to support nested tags? Kind of like a recursive approach? I could be completely off 😂

Reply:
eat is a remarkjs convention - essentially we are iterating through the `value`, and when we find something we want to parse and redact, we `eat` that part of the value and replace it with a redaction node.

Reply:
Ahh makes sense. Thanks for the explanation!